apps/src/lib/levelbuilder/announcementsEditor/AnnouncementsEditor.jsx

```diff
@@ -18,8 +18,7 @@ const styles = {
 export default class AnnouncementsEditor extends Component {
   static propTypes = {
     defaultAnnouncements: PropTypes.arrayOf(announcementShape),
-    inputStyle: PropTypes.object.isRequired,
-    curriculumObject: PropTypes.string
```

Comment:
What was `curriculumObject` - is it just not needed anymore?

Reply:
It's not needed anymore. It was just being used for the name of the announcements so you could write `${curriculumObject}_announcements`